java/semantickernel-api/src/main/java/com/microsoft/semantickernel/templateengine/semantickernel/blocks/CodeBlock.java

```diff
@@ -120,7 +121,7 @@ private <T> Mono<ContextVariable<T>> renderFunctionCallAsync(
             //Cloning the original arguments to avoid side effects - arguments added to the original arguments collection as a result of rendering template variables.
             arguments = this.enrichFunctionArguments(kernel, fBlock,
                 arguments ==
-                    null ? new DefaultKernelArguments() : new DefaultKernelArguments(arguments));
+                    null ? new DefaultKernelArguments() : new DefaultKernelArguments(arguments, new HashMap<>()));
```

Comment:
Is the hash map mutated downstream? If not, maybe use Collections.emptyMap() here?